### PR TITLE
[Spot 225] Updates to ODM documentation

### PR DESCRIPTION
**Pull request by mhicks-cloudera-com**
The document was enhanced as follows:

- added additional attributes to the event schema to enhance coverage for antivirus events
- added an additional "vulnerability" object/schema to store corresponding context information
- fixed various spelling errors and typos

### CODE REVIEW

**Comment by vgonzale78** [comment]
Hi @mhicks-cloudera-com 

There are references to 2 images that seems to be broken, 

Line 77 --> image alt text
LIne 3002  --> image alt text

File: docs/open-data-model.md [hunk 1]
```diff
@@ -0,0 +1,3310 @@
+Apache Spot Open Data Model (ODM)
```
Comment (natedogs911) [inline docs/open-data-model.md]: needs to include (incubating)

**Comment by natedogs911** [comment]
Approving with a couple of requests,

+1

**Comment by jarcec** [comment]
+1 From my side as well :)

**Comment by natedogs911** [comment]
having trouble with the merge, should be merging shortly

**Pull request merged**